documentation/devel/git-hooks.md

```diff
@@ -0,0 +1,27 @@
+# Developer documentation: how to install (software for) git hooks
+
+You first need to have Python and pip
+
+## Install [pre-commit](https://pre-commit.com)
+See https://pre-commit.com/#install but the following might work.
+
+    sudo -H pip install pre-commit
```

Comment:
```suggestion
    pip install pre-commit
```

Reply:
I haven't tried this but put this in because it isn't mentioned on https://pre-commit.com/#install. But if you tell me it works, that'd be great.

Reply:
`sudo` (even with `-H`) is actively discouraged with `pip`. `--user` will definitely work but is neither required nor recommended in virtualenv/conda, which is why it's best to add it as a note rather than by default.